recipes/gbmunge/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "gbmunge" %}
+{% set version = "2018.07.06" %}
+{% set sha256 = "1297e73be293a7b98bc080faeb191737b0ad4cc5ca3a9ec2564c9b82e95cc014" %}
+{% set git_rev = "4117e04da03a0dad41417397c623dc5e995794df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sdwfrost/gbmunge/archive/{{ git_rev }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - makefile.patch
```

Comment:
Just a suggestion, but if you do the `cp` in the `build.sh` and use `make CC=$CC` you can probably get away without the patch file.